finance/fundraising.md

```diff
@@ -0,0 +1,6 @@
+# Fundraising and philanthropy pipeline
+
+We have a lightweight CRM and pipeline for raising funds through philanthropic gifts.
+This is under heavy development, so all information is currently contained in the documebt below:
+
+[Fundraising and philanthropy CRM / pipeline guide](https://docs.google.com/document/d/1QaIlqtQnCdXtshX8pZTP7tquc4MvYLSY6WiBZHqrL_c/edit?usp=sharing).
```

Comment:
```suggestion
We have a lightweight CRM and pipeline for raising funds through philanthropic gifts.
The document below describes the way this worked as of August 2024. It has since been integrated with a broader Sales CRM that was inspired by this one.
We need to update the information in the Google Doc accordingly, but leave it here for now because there is still a lot of useful information in it.

[Fundraising and philanthropy CRM / pipeline guide](https://docs.google.com/document/d/1QaIlqtQnCdXtshX8pZTP7tquc4MvYLSY6WiBZHqrL_c/edit?usp=sharing).
```